### PR TITLE
Cookie initialy baked by NetworkToCode Cookie Drift Manager Tool

### PR DESCRIPTION
**Pull request by snaselj**
Template:

```
{
    "template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
    "dir": "nautobot-app",
    "ref": "develop",
    "path": null
}
```

Cookie:

```
{
    "remote": "https://github.com/nautobot/nautobot-plugin-capacity-metrics.git",
    "path": "/opt/ntc/drift-manager/outputs/nautobot-plugin-capacity-metrics",
    "repository_path": "/opt/ntc/drift-manager/outputs/nautobot-plugin-capacity-metrics",
    "dir": "",
    "branch_prefix": "drift-manager",
    "context": {
        "codeowner_github_usernames": "@nautobot/po-tl-telemetry",
        "full_name": "Network to Code, LLC",
        "email": "opensource@networktocode.com",
        "github_org": "nautobot",
        "plugin_name": "nautobot_capacity_metrics",
        "verbose_name": "Metrics & Monitoring Extension Plugin",
        "plugin_slug": "nautobot-capacity-metrics",
        "project_slug": "nautobot-plugin-capacity-metrics",
        "repo_url": "https://github.com/nautobot/nautobot-plugin-capacity-metrics",
        "base_url": "capacity-metrics",
        "min_nautobot_version": "2.0.0",
        "max_nautobot_version": "2.9999",
        "camel_name": "NautobotCapacityMetrics",
        "project_short_description": "Plugin to improve the instrumentation of Nautobot and expose additional metrics (Application Metrics, RQ Worker).",
        "model_class_name": "None",
        "open_source_license": "Apache-2.0",
        "docs_base_url": "https://docs.nautobot.com",
        "docs_app_url": "https://docs.nautobot.com/projects/capacity-metrics/en/latest",
        "_template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
        "_output_dir": "/opt/ntc/drift-manager/outputs",
        "_repo_dir": "/home/jsn/.cookiecutters/cookiecutter-nautobot-app/nautobot-app",
        "_checkout": "develop"
    },
    "base_branch": "develop",
    "remote_name": "origin",
    "pull_request_strategy": "PullRequestStrategy.CREATE",
    "post_actions": [
        "PostAction.BLACK"
    ],
    "baked_commit_ref": "",
    "draft": true
}
```

CLI Arguments:

```
{
    "cookie_dir": "",
    "input": false,
    "json_filename": "setup-cookie-capacity-metrics.json",
    "output_dir": "./outputs",
    "push": true,
    "template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
    "template_dir": "nautobot-app",
    "template_ref": "develop",
    "pull_request": null,
    "post_action": [
        "black"
    ],
    "disable_post_actions": false,
    "draft": true
}
```

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -22,131 +33,204 @@ jobs:
...
             nautobot-version: "2.0.0"
-          - python-version: "3.11"
-            db-backend: "postgresql"
-            nautobot-version: "stable"
-          - python-version: "3.11"
-            db-backend: "mysql"
-            nautobot-version: "stable"
+          # - python-version: "3.11"
+          #   db-backend: "mysql"
+          #   nautobot-version: "stable"
```
Comment (snaselj) [inline .github/workflows/ci.yml:187]: MYSQL unit tests are failing, in the current `develop` branch, it was using PostgreSQL tests, even when `db-backend == 'mysql'`.

**Comment by snaselj** [comment]
This PR is ready for review.

cc.: @whitej6

File: docs/nautobot_grafana_dashboard.json
Comment (whitej6) [inline docs/nautobot_grafana_dashboard.json:1]: Need to verify references of this file and if it's just docs related

File: docs/images/nautobot_grafana_dashboard.png
Comment (whitej6) [inline docs/images/nautobot_grafana_dashboard.png:1]: Need to verify all references point to new location

**Review by whitej6 — COMMENTED** [review]
Need to validate 2 references before files are moved, & @cmsirbu to thumbs up README.md/index.md shuffling of information

File: docs/index.md
[the remark this answers is not in the document]
Comment (snaselj) [reply docs/index.md:1]: `README.md` was just a symlink to `docs/index.md` before. I moved everything to `README.md` to be aligned with the template.

**Comment by snaselj** [comment]
Already merged in https://github.com/nautobot/nautobot-app-capacity-metrics/pull/73

**Pull request closed without merging**